# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../fget/hotfix.patch
# Copyright (C) 2019 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- fget/fget.c.vanilla	2019-02-07 10:17:45.512424271 +0000
+++ fget/fget.c	2019-02-07 10:18:13.596423951 +0000
@@ -150,15 +150,15 @@
   stralloc_cats(request,"\r\n\r\n");
 }
 
-inline int isdigit(char c) {
+int isdigit(char c) {
   return (c>='0' && c<='9');
 }
 
-inline int isresponsecode(char *s) {
+int isresponsecode(char *s) {
   return (isdigit(s[0]) && isdigit(s[1]) && isdigit(s[2]) && (s[3]==' ' || s[3]=='-'));
 }
 
-inline int iscrlf(char *s) {
+int iscrlf(char *s) {
   return *s=='\r' && s[1]=='\n';
 }
 
--- fget/readwrite.h.vanilla	2019-02-07 10:27:17.936417746 +0000
+++ fget/readwrite.h	2019-02-07 10:28:03.400417227 +0000
@@ -1,8 +1,12 @@
 #ifndef READWRITE_H
 #define READWRITE_H
 
+#include <unistd.h>
+
+#if 0
 extern int read(int fd,void *buf,unsigned int count);
 extern int write(int fd,const void *buf,unsigned int count);
 extern int close(int fd);
+#endif
 
 #endif
